### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by paulciorogar**

### CODE REVIEW

**Review by dasilvaluis — APPROVED** [review]
Good initial setup :+1:

**Review by alexandru-calinoiu — APPROVED** [review]
Looks good but there is a lot more stuff here then an initial setup

For an ideal initial setup I would have had the services return minimal mock data

**Comment by dasilvaluis** [comment]
As a minor comment, there is still this test file `src/App.test.js` about the initial scaffold text. Maybe it should be removed entirely

File: devServer/server.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const jsonServer = require('json-server');
+const server = jsonServer.create();
+const middlewares = jsonServer.defaults();
+const path = require('path');
+const router = jsonServer.router(path.join(__dirname, 'db.json'));
+
+server.use(middlewares);
```
Comment (alexandru-calinoiu) [inline devServer/server.js:7]: Can you describe the rationale behind the dev webserver please? I think I understand it's usage but I wonder if the benefits outweigh the costs.

Comment (paulciorogar) [reply devServer/server.js:7]: I want a way to test different scenarios (errors) that I could not do with the provided resource

**Comment by paulciorogar** [comment]
> As a minor comment, there is still this test file `src/App.test.js` about the initial scaffold text. Maybe it should be removed entirely

I missed that, thank you

File: devServer/server.js (hunk 1 above)
Comment (alexandru-calinoiu) [reply devServer/server.js:7]: Isn't it enough to test them in unit tests?

Comment (paulciorogar) [reply devServer/server.js:7]: I guess it's enough. I'll try working without it.

**Pull request merged**